GuildedRoseLLCUITests/GuildedRoseLLCUITests.swift

```diff
@@ -1,8 +1,30 @@
-//
-//  GuildedRoseLLCUITests.swift
-//  GuildedRoseLLCUITests
-//
-//  Created by Ethan Whaley on 5/25/21.
-//
-
+import Quick
+import Nimble
 import XCTest
+import GuildedRoseLLC
+
+class GuildedRoseLLCSpec: QuickSpec {
+    override func spec() {
+        var app: XCUIApplication!
+        beforeEach {
+            app = XCUIApplication()
+            app.launch()
+        }
+        describe("a sample test") {
```

Comment:
Request for Change: You can get rid of this test since the one after passes.

Reply:
+1